src/software/ai/navigator/path_planner/path_planner_performance_test.cpp

```diff
@@ -0,0 +1,224 @@
+    obstacle_factory.createRobotObstacle({-1.5, 0}),
+    obstacle_factory.createRobotObstacle({-1.5, 0.5}),
+    obstacle_factory.createRobotObstacle({-1.5, 1.0}),
+    obstacle_factory.createRobotObstacle({-1.5, 1.5}),
+    obstacle_factory.createRobotObstacle({1.5, 0}),
+    obstacle_factory.createRobotObstacle({1.5, 0.5}),
+    obstacle_factory.createRobotObstacle({1.5, 1.0}),
+    obstacle_factory.createRobotObstacle({1.5, 1.5}),
+};
+
+std::vector<ObstaclePtr> obstacles_30 = {
+    obstacle_factory.createRobotObstacle({0, 0}),
+    obstacle_factory.createRobotObstacle({0, 0.5}),
+    obstacle_factory.createRobotObstacle({0, 1.0}),
+    obstacle_factory.createRobotObstacle({0, 1.5}),
+    obstacle_factory.createRobotObstacle({-0.5, 0}),
+    obstacle_factory.createRobotObstacle({-0.5, 0.5}),
+    obstacle_factory.createRobotObstacle({-0.5, 1.0}),
+    obstacle_factory.createRobotObstacle({-0.5, 1.5}),
+    obstacle_factory.createRobotObstacle({0.5, 0}),
+    obstacle_factory.createRobotObstacle({0.5, 0.5}),
+    obstacle_factory.createRobotObstacle({0.5, 1.0}),
+    obstacle_factory.createRobotObstacle({0.5, 1.5}),
+    obstacle_factory.createRobotObstacle({1, 0}),
+    obstacle_factory.createRobotObstacle({1, 0.5}),
+    obstacle_factory.createRobotObstacle({1, 1.0}),
+    obstacle_factory.createRobotObstacle({1, 1.5}),
+    obstacle_factory.createRobotObstacle({-1.5, 0}),
+    obstacle_factory.createRobotObstacle({-1.5, 0.5}),
+    obstacle_factory.createRobotObstacle({-1.5, 1.0}),
+    obstacle_factory.createRobotObstacle({-1.5, 1.5}),
+    obstacle_factory.createRobotObstacle({1.5, 0}),
+    obstacle_factory.createRobotObstacle({1.5, 0.5}),
+    obstacle_factory.createRobotObstacle({1.5, 1.0}),
+    obstacle_factory.createRobotObstacle({1.5, 1.5}),
+    obstacle_factory.createRobotObstacle({2, 0}),
+    obstacle_factory.createRobotObstacle({2, 0.5}),
+    obstacle_factory.createRobotObstacle({2, 1.0}),
+    obstacle_factory.createRobotObstacle({2, 1.5}),
+    obstacle_factory.createRobotObstacle({-2.5, 0}),
+    obstacle_factory.createRobotObstacle({-2.5, 0.5})};
+
+std::vector<PlannerTestCase> test_cases = {
+    {.name           = "Empty small area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = Rectangle({-2, -2}, {2, 2}),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty medium area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = Rectangle({-3, -3}, {3, 3}),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty divB area short path",
+     .start          = Point(-1, 0),
+     .end            = Point(1, 0),
+     .navigable_area = ::Test::TestUtil::createSSLDivBField().fieldBoundary(),
+     .obstacles      = {},
+     .num_iterations = 10},
+
+    {.name           = "Empty divB area long path",
```

Comment:
nit: we use `divB` and `full field` interchangeably in the naming. Can we choose one to be consistent?

Reply:
choosing divB